curationexperts/t3
# Display the favicon in tabs and previews

This commit updates the main layout file to use the custom favicon from the theme instead of a default blacklight favicon.

The favicons are also displayed next to the theme label in theme previews.

> TECH NOTE:
> In tests, attachments can be created in a different transaction
> than the main Theme causing the attachment to get rolled back, while
> the theme record pointing to it remains persisted. Generally, we want
> to test for
> ```
>   return if favicon.attached?
> ```
> but the attachment state can get out of sync because of test database
> example rollbacks, so we need to confirm that the attachment is still
> actually present, so we can try this instead:
> ```
>  return if favicon.try(:signed_id)
> ```
>Which fails unless both sides of the attachment relationship are still
> valid.
